team-01/src/main/java/ubc/cosc322/AmazonsLocalBoard.java

```diff
@@ -24,25 +24,27 @@
 
 	public void setState(ArrayList<Integer> state) {
 		this.state = state;
-		printState();
+	}
+
+	public void updateState(AmazonsAction action) {
+		updateState(action.queenCurrent, action.queenTarget, action.arrowTarget);
 	}
 
 	public void updateState(ArrayList<Integer> queenCurrent, ArrayList<Integer> queenTarget, ArrayList<Integer> arrowTarget) {
 		int playerColor = getPositionValue(queenCurrent);
 		setPositionValue(queenCurrent, 0);
 		setPositionValue(queenTarget, playerColor);
 		setPositionValue(arrowTarget, 3);
-		printState();
-	}
-
-	public int getPositionValue(ArrayList<Integer> position) {
-		return state.get(getIndex(position));
 	}
 
 	public int getPositionValue(int x, int y) {
 		return getPositionValue(new ArrayList<>(Arrays.asList(x, y)));
 	}
 
+	public int getPositionValue(ArrayList<Integer> position) {
+		return state.get(getIndex(position));
+	}
```

Comment:
I find getValueAt() clearer, but I'm okay with this too.

Reply:
I didn't change this code, I just moved it (to match the format of my new method). However, feel free to refactor some code in LocalBoard in another PR if you would like to change the wording.

Reply:
Gotcha. Yeah sounds good.